cleanslate/templates/registration/login.html

```diff
@@ -1,9 +1,23 @@
 {% extends "base.html" %}
 
+{% block title %}<title>Login</title>{% endblock %}
+
+{% block extra_head %}
+    {% load static %}
+        <head>
+            <link href="{% static 'css/login.css' %}" rel="stylesheet" type="text/css" />
+        </head>
+{% endblock %}
+
 {% block content %}
-<div class = "row">
-<div  class = "col offset-s5" style = "text-align: center">
 
+
+<div class="form">
+              
+<div id="login">   
```

Comment:
Just a stylistic comment, but could you take a minute to fix the indentation and spacing on this page